layouts/guides.tsx

```diff
@@ -6,10 +6,14 @@ import FooterCTA from "../components/base/FooterCTA";
 import { breakUp } from "../components/breakpoints";
 
 const StyledPagination = styled.div`
-  margin: 20px;
+  padding: 20px;
 
   @media (${breakUp("md")}) {
-    margin: 40px;
+    padding: 40px;
+  }
+
+  @media (${breakUp("lg")}) {
+    width: 70%;
```

Comment:
The main point of this PR is to allow these layouts, and the components in them, to not be bound by width the mdx content has to accommodate the "on this page" component. I.e. Components like the FooterCTA here can take up the full width of the content area